src/Charcoal/Email/Exception/EmailNotSentException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Charcoal\Email\Exception;
+
+use Exception;
```

Comment:
```suggestion
use RuntimeException;
```
Related to a comment below.